Exec/ABL/inputs_GABLS1_ysu

```diff
@@ -66,15 +66,11 @@ erf.abl_driver_type = "GeostrophicWind"
 erf.abl_geo_wind = 8.0 0.0 0.0
 
 # Turbulence closure
-erf.les_type        = "None"
-
-# NOT USED
-#erf.rho0_trans      = 1.3223 # from Cuxart et al. 2006
-#erf.theta_ref       = 263.5 # from Cuxart et al. 2006
-
+erf.les_type    = "None"
 erf.pbl_type    = "YSU"
 
 # Initial conditions from Beare et al. 2006
 prob.KE_0            = 0.4 # [m2/s2]
 prob.KE_decay_height = 250. # [m]
-prob.KE_decay_order  = 1
+prob.KE_decay_order  = 3
```

Comment:
Actually can you just delete the prob.KE_0 inputs for the YSU input file? These go unused because it doesn't use either QKE or TKE.

Reply:
Will do